### ISSUE TITLE
Can't build: 'ObjectiveGit/ObjectiveGit.h' file not found

### ISSUE THREAD

**Issue by raxod502** [mssun/passforios#45]
I followed the steps in the **Build** section of the README. Upon building the project with Xcode, I get the following errors:

```
/Users/raxod502/Desktop/Code/iOS/passforios/pass/Helpers/Objective-CBridgingHeader.h:13:9: error: 'ObjectiveGit/ObjectiveGit.h' file not found
#import <ObjectiveGit/ObjectiveGit.h>
        ^
<unknown>:0: error: failed to import bridging header '/Users/raxod502/Desktop/Code/iOS/passforios/pass/Helpers/Objective-CBridgingHeader.h'
```

This seems to be because something is wrong with CocoaPods. In particular, the Internet suggests that a file called `pods.xcconfig` is supposed to be generated (it isn't), and that it can cause problems if it is missing.

I don't really know how to debug this further. Any tips would be appreciated—I'd love to get this app working on my phone (typing in my `pass`-generated Apple ID is getting pretty old).

**Comment by mssun** [comment]
I tried to clone and build the project from scratch, but not reproduced your issue. Can you try to follow the instruction and do it again?

I guess you may forget `pod install`?

**Comment by raxod502** [comment]
Thanks. I definitely didn't forget to do `pod install`. In fact, I did a number of `pod`-related commands to uninstall and reinstall things, none of which fixed the issue.

But today I cloned a fresh copy of the project and it built fine. I'm 99% sure this is due to Xcode doing some stupid caching thing, as there were a few settings I had to change last time to fix unrelated build errors (e.g. changing the team to my personal team) that I *didn't* have to change this time around.

Either that, or I'm insane. Sorry :P 

In any case, [here](https://gist.github.com/raxod502/501a9c933f9b77dd611982519b5b4c87)'s the build log in case that's useful to anyone in the future.
